### DISCUSSION TITLE
How to update from 0.9.4?

### DISCUSSION THREAD

**Discussion by niemand0** [Homebrew/discussions#2894] [Getting started] [answered]
I ran brew update, and...
```
$ brew update
error: error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version while accessing https://github.com/mxcl/homebrew.git/info/refs

fatal: HTTP request failed
Error: Failure while executing: git pull -q origin refs/heads/master:refs/remotes/origin/master
```
So how do I update to the latest?
Thanks.

Running on Snow Leopard Server.

**Comment by gromgit** [comment, marked as answer]
The latest Homebrew absolutely will not run on Snow Leopard. The earliest macOS version that's even referenced in the current Homebrew code is Yosemite, so you'd have to upgrade your OS to at least that version in order to even _try_ building stuff from source.

And even if you could do the upgrade, building from source may well fail anyway for many formulae.

Sorry.
